config/locales/hyrax.en.yml

```diff
@@ -257,13 +257,21 @@ en:
       workflow_roles:
         header: Workflow Roles
         index:
+          current_group_roles: Current Group Roles
```

Comment:
remove line 260 for this and all other languages

Reply:
current_group_roles: Current Group Roles has been deleted for all langauges